pkg/controller/machine/machine_controller.go

```diff
@@ -305,3 +340,27 @@ func (r *ReconcileMachine) deleteNode(ctx context.Context, cluster *clusterv1.Cl
 
 	return corev1Remote.Nodes().Delete(name, &metav1.DeleteOptions{})
 }
+
+// getMachines returns all of the Machine objects that belong to the same
+// cluster as the provided Machine
+func (r *ReconcileMachine) getMachines(ctx context.Context, machine *clusterv1.Machine) ([]*clusterv1.Machine, error) {
+	clusterName := machine.Labels[clusterv1.MachineClusterLabelName]
+	if clusterName == "" {
+		return nil, nil
+	}
+
+	mlist := &clusterv1.MachineList{}
```

Comment:
We try to avoid abbreviations in names - could this be named `machineList`?

Reply:
Hi @ncdc,

Could be :) FWIW, I copied this from the controller-api package. Their convention for item lists was `Xlist`. Happy to make it whatever you want though.